cloudwatch_metrics_common/include/cloudwatch_metrics_common/utils/metric_object.h

```diff
@@ -46,17 +47,53 @@ static std::unordered_map<std::string, Aws::CloudWatch::Model ::StandardUnit> un
   {"", Aws::CloudWatch::Model::StandardUnit::None},
 };
 
+enum class StatisticValuesType
+{
+  MINIMUM,
+  MAXIMUM,
+  SUM,
+  SAMPLE_COUNT
+};
+
 /**
  * Wrapper object for the AWS specific Aws::CloudWatch::Model::MetricDatum. This object is meant to be constructed from
  * userland provided metric data instead of using the AWS SKD specific object.
```

Comment:
nit: typo in the comment

```suggestion
 * userland provided metric data instead of using the AWS SDK specific object.
```

Reply:
Done.